addons/binding/org.openhab.binding.harmonyhub/.classpath

```diff
@@ -16,5 +16,6 @@
 	<classpathentry kind="lib" path="lib/smack-tcp-4.2.0.jar"/>
 	<classpathentry kind="lib" path="lib/xpp3-1.1.4c.jar"/>
 	<classpathentry kind="lib" path="lib/harmony-java-client-1.2.2.jar"/>
+	<classpathentry kind="lib" path="lib/harmony-client-1.0-SNAPSHOT.jar"/>
```

Comment:
A released version would be nice. :-)
Can you also update the NOTICE file for the library change?

Reply:
👍